Scripts/options.gd

```diff
@@ -12,6 +12,12 @@ func _ready():
 func _on_reset_btn_pressed():
 	controller.reset()
 
+func _on_enforce_8bit_btn_toggled( pressed ):
+	if pressed:
+		controller.setMaxValue(256)
```

Comment:
`maxValue` is exclusive, so its not really the maximum value. In my recommendation, I had the max value be set to `255` instead of `256`, but I see why that makes things more complicated and could confuse. Perhaps it needs a better name. `valueCeiling`? `dataCeil`? idk.